frontend/src/metabase-lib/metadata.ts

```diff
@@ -33,6 +33,7 @@ export function metadataProvider(
   databaseId: DatabaseId,
   metadata: Metadata,
 ): MetadataProvider {
+  console.log("Initializing new metadata provider for metadata", metadata);
```

Comment:
Unrelated but I added this to make it easier to debug metadata issues like #32511